medyator/kink/kink_service_provider.py

```
@@ -1,30 +1,67 @@
-from typing import Union, cast
+from typing import Type, Union, cast
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)